framework/core/js/src/common/Application.tsx

```diff
@@ -248,6 +249,8 @@ export default class Application {
 
   allowUserColorScheme!: boolean;
 
+  refs: Record<string, unknown> = {};
```

Comment:
Let's directly add the value here instead of calling a function. The function isn't necessary at this time 👍